osu.Game/Graphics/UserInterface/Nub.cs

```diff
@@ -163,14 +165,20 @@ public Color4 GlowColour
 
         private void onCurrentValueChanged(ValueChangedEvent<bool> filled)
         {
-            fill.FadeTo(filled.NewValue ? 1 : 0, 200, Easing.OutQuint);
+            const double duration = 200;
+
+            fill.FadeTo(filled.NewValue ? 1 : 0, duration, Easing.OutQuint);
 
             if (filled.NewValue)
-                main.ResizeWidthTo(1, animate_in_duration, Easing.OutElasticHalf);
+            {
+                main.ResizeWidthTo(1, duration, Easing.OutElasticHalf);
+                main.TransformTo(nameof(BorderThickness), filled.NewValue ? 8.5f : border_width, duration, Easing.OutElasticHalf);
+            }
             else
-                main.ResizeWidthTo(0.9f, animate_out_duration, Easing.OutElastic);
-
-            main.TransformTo(nameof(BorderThickness), filled.NewValue ? 8.5f : border_width, 200, Easing.OutQuint);
+            {
+                main.ResizeWidthTo(0.75f, duration, Easing.OutQuint);
+                main.TransformTo(nameof(BorderThickness), border_width, duration, Easing.OutQuint);
```

Comment:
Just to confirm - the easing changes from `OutElastic(Half)` here were intended?

Reply:
Yeah, this was intentional. While `OutElastic` feels quite good here, the border fill does a weird pivot at the end which looks weird so I switched this away.

More interestingly, in the latest design there is no horizontal size adjust, so this may just go away at some point...